data/pages.yml

```diff
@@ -12,6 +12,8 @@
       url: "index"
     - title: "Your First Ember App"
       url: "first-app"
+    - title: "Glossary"
+      url: "glossary"
     - title: "Core Concepts"
       url: "core-concepts"
```

Comment:
I really like the idea of a Glossary, but wonder if putting it here in the Getting Started section is going to stop people cold (especially when we rearrange things when we merge in the tutorial).  Could we find a different spot to park the Glossary link and instead link to the appropriate section as needed when we first introduce a new concept in the tutorial / guides?

Seems like that would allow newcomers to catch up where needed without them needing to read the entire page as part of an intro to Ember ...